src/ResourceManager/DataFactories/Commands.DataFactories/Commands.DataFactories.Netcore.csproj

```diff
@@ -29,7 +29,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.DataFactories" Version="4.13.3" />
-    <PackageReference Include="Microsoft.DataTransfer.Gateway.Encryption" Version="3.4.6604.1" />
+    <PackageReference Include="Microsoft.DataTransfer.Gateway.Encryption" Version="3.11.6886.9" />
+    <PackageReference Include="System.IO.FileSystem.AccessControl" Version="4.5.0" />
+    <PackageReference Include="WindowsAzure.Storage" Version="9.3.0" />
```

Comment:
You'll need to add these dlls in the `RequiredAssemblies` section of `Az.DataFactories.ps1`. The management dll and the encryption dll are already listed. You'll need the AccessControl and Storage dlls there too.

Reply:
Done.